Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -105,8 +105,17 @@ file that was distributed with this source code.
             {{ form_widget(form) }}
         </span>
 
-        <div class="container sonata-ba-modal sonata-ba-modal-edit-one-to-one" style="display: none" id="field_dialog_{{ id }}">
-
+        <div class="modal fade" id="field_dialog_{{ id }}" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
```

Comment:
This is also fixed? @enleur AFAIK this template was not moved to SonataAdmin

Reply:
Yeah, my bad

Reply:
Feel free to create a new PR for this part, so we can close this one